ssm_jax/hmm/demos/poisson_hmm_earthquakes.py

```diff
@@ -11,187 +11,176 @@
 import matplotlib.pyplot as plt
 from ssm_jax.hmm.models.poisson_hmm import PoissonHMM
 
-
-def main(test_mode=False):
-    # earthquake data from http://earthquake.usgs.gov/
-    earthquakes = jnp.array([
-        13,
-        14,
-        8,
-        10,
-        16,
-        26,
-        32,
-        27,
-        18,
-        32,
-        36,
-        24,
-        22,
-        23,
-        22,
-        18,
-        25,
-        21,
-        21,
-        14,
-        8,
-        11,
-        14,
-        23,
-        18,
-        17,
-        19,
-        20,
-        22,
-        19,
-        13,
-        26,
-        13,
-        14,
-        22,
-        24,
-        21,
-        22,
-        26,
-        21,
-        23,
-        24,
-        27,
-        41,
-        31,
-        27,
-        35,
-        26,
-        28,
-        36,
-        39,
-        21,
-        17,
-        22,
-        17,
-        19,
-        15,
-        34,
-        10,
-        15,
-        22,
-        18,
-        15,
-        20,
-        15,
-        22,
-        19,
-        16,
-        30,
-        27,
-        29,
-        23,
-        20,
-        16,
-        21,
-        21,
-        25,
-        16,
-        18,
-        15,
-        18,
-        14,
-        10,
-        15,
-        8,
-        15,
-        6,
-        11,
-        8,
-        7,
-        18,
-        16,
-        13,
-        12,
-        13,
-        20,
-        15,
-        16,
-        12,
-        18,
-        15,
-        16,
-        13,
-        15,
-        16,
-        11,
-        11,
-    ])
-
-    # Plot the sampled data
-    fig, ax = plt.subplots()
-    ax.plot(earthquakes, ".-", ms=6, mfc="orange", alpha=0.7)
-    ax.set_xticks(range(0, earthquakes.size, 10))
-    ax.set_xticklabels(range(1906, 2007, 10))
-    ax.set_xlabel("Year")
-    ax.set_ylabel("Count")
-    fig.show()
-
+# earthquake data from http://earthquake.usgs.gov/
+EARTHQUAKES = jnp.array([
```

Comment:
we should put all the data on one line and turn of black line length / format checking using `# fmt: off`